### PR TITLE
Added HUGGO and AIGGO datasets

### PR DESCRIPTION
**Pull request by jaeltan**
# Description

Minor changes to the package including replacing the GNEVAR and GNEVAR_TXT datasets with AIGGO and HUGGO datasets respectively. These datasets are stored in the agreements database in the package. The changes are made as part of the restructuring of the "many" packages. HUGGO refers to hand coded (or verified) data, while AIGGO refers to programatically coded data.

## Data
* Added `HUGGO` and `AIGGO` datasets to the `agreements` database.
  * `HUGGO` dataset contains manually coded or verified data and treaty texts, replacing the `GNEVAR_TXT` dataset in the texts database.
  * `AIGGO` dataset contains programatically coded information on text treaties such as accession conditions and procedures that were coded using `manypkgs::code_accession()`, replacing the `GNEVAR` dataset.
* Deleted texts database since all treaty texts are now stored in `HUGGO`.
* Treaty texts were cleaned again using the updated `manypkgs::standardise_treaty_text()`.

# Checklist:

- [x] PR form
  - [x] I have given this pull request an informative title
  - [x] Description above itemizes changes under subtitles, e.g. "## Collection""
  - [x] Any closed, fixed, or related issues are referenced and explained in the description above, e.g. "Fixed #0 by adding A"
  - [x] Package builds on my OS without issues
- [x] PR checks all pass for latest commit
  - [x] Package builds on Mac
  - [x] Package builds on Windows
  - [x] Package builds on Linux
- [x] Documentation
  - [x] Any new or modified functions or data have roxygen style documentation in their .R scripts
  - [x] Any longer functions are commented inline so that it is easier to debug in the future
  - [x] PR description above and the NEWS.md file are aligned
  - [x] DESCRIPTION file version is bumped by the appropriate increment (major, minor, patch)

### CODE REVIEW

File: data-raw/agreements/AIGGO/prepare-AIGGO.R [hunk 1]
```diff
@@ -47,7 +45,7 @@ GNEVAR$dates <- lapply(GNEVAR$TreatyText, function(s){
   s <- paste0(unlist(s), collapse = "")
 })
 
-GNEVAR <- GNEVAR %>%
+AIGGO <- AIGGO %>%
   dplyr::relocate(manyID, Title, Beg, Signature, dates) %>%
```
Comment (henriquesposito) [inline data-raw/agreements/AIGGO/prepare-AIGGO.R:49]: Could you please remove other variables inherited from HUGGO but that are not pertinent here (e.g. TreatyText)?

Comment (henriquesposito) [reply data-raw/agreements/AIGGO/prepare-AIGGO.R:49]: Could you also make sure that NAs are properly introduced across columns (see the latest changes in manyenviron for reference) as well as that there are no NAs/missing observations when it comes to manyID or agreements titles?

File: data-raw/agreements/HUGGO/prepare-HUGGO.R [hunk 2]
```diff
@@ -203,48 +196,48 @@ REM_TXT$Text <- lapply(REM_TXT$url, function(x) {
...
 
-GNEVAR_TXT <- subset(GNEVAR_TXT,
-                     subset = !duplicated(GNEVAR_TXT[, c(1,2,3,4,5,6,7,8,9,10,11,12,13)]))
+HUGGO <- subset(HUGGO,
+                     subset = !duplicated(HUGGO[, c(1,2,3,4,5,6,7,8,9,10,11,12,13)]))
 
-GNEVAR_TXT <- GNEVAR_TXT %>%
-  dplyr::relocate(manyID, Title, Beg, Signature, Force, totaID, gptadID, destaID,
-                  labptaID, trendID, treatyID)
+HUGGO <- HUGGO %>%
```
Comment (henriquesposito) [inline data-raw/agreements/HUGGO/prepare-HUGGO.R:231]: Could you please make sure that any text variables are correctly named (i.e. contain the word text in names)?

Could you also make sure that NAs are properly introduced across columns (see the latest changes in manyenviron for reference) as well as that there are no NAs/missing observations when it comes to manyIDs?

Comment (jaeltan) [reply data-raw/agreements/HUGGO/prepare-HUGGO.R:228]: Thanks for the feedback @henriquesposito . The only text variable in HUGGO is TreatyText and I've already removed all the other extra variables from AIGGO. There are no NAs for manyIDs or agreement titles.

**Review by henriquesposito — APPROVED** [review]
Thank you for the great work @jaeltan !

Before you merge branches, please just make sure the workflows are up to date. As well, the default bump in the push releases workflow is "patch"; this sometimes overwrites the commit tags (i.e. you may want to remove this line from the push release workflows for now so that the minor bump works).

File: DESCRIPTION [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Package: manytrade
 Title: Trade Agreements for manydata
-Version: 0.1.2
-Date: 2022-07-13
+Version: 0.2.0
```
Comment (jhollway) [inline DESCRIPTION:3]: I think this is a #minor update (not #patch), just need to make sure it is all aligned.

**Review by jhollway — APPROVED** [review]

**Pull request merged**